roles/add-sonar-plugin/tests/test.yml

```diff
@@ -6,5 +6,5 @@
     vars:
       plugin_url: https://github.com/SonarOpenCommunity/sonar-cxx/releases/download/cxx-1.2.2/sonar-cxx-plugin-1.2.2.1653.jar
       install_location: /opt/sonarqube/extensions/plugins
-      namespace: deven-role-test
+      namespace: deven-ci-cd
```

Comment:
probably should make this more generic than `deven`(?)

Reply:
Updated